css/styles.css

```diff
@@ -183,6 +187,35 @@ nav li {
 	background-image: url("../IMG/comingSoon.jpg");
 }
 
+.contactMe {
+	overflow: auto;
```

Comment:
Might not be required - turning off in inspector didn't do anything.

Reply:
removed